2018/Users-Meeting/Workshops/Metadata-Import/slides/index.html

```diff
@@ -0,0 +1,236 @@
+                        <li>without Pixels files, less data duplication</li>
+                        <li>original data visible in server filesystem</li>
+                    <ul>
+                        <li>where filesets live</li>
+                        <li>each Fileset in its own subdirectory</li>
+                    </ul>
+                    </aside>
+                </section>
+                <section>
+                    <h2>For Demo, user importing is...</h2>
+                    <ul>
+                        <li>A Facility Manager: <i>importer1</i> i.e. an OMERO user with some restricted
+                            <a href="https://docs.openmicroscopy.org/latest/omero/sysadmins/restricted-admins.html">admin privileges</a></li>
+                        <li>who has <b>Write</b> access to the <a href="https://docs.openmicroscopy.org/latest/omero/developers/Server/FS.html">ManagedRepository</a></li>
+
+                    </ul>
+                </section>
+                <section>
+                    <h2>In-place import: symlink</h2>
+                    <code>
+                        $ bin/omero import path/to/image --transfer=ln_s
+                    </code>
+                    <img src="images/inplace-ln_s.png" width="90%"/>
+                </section>
+                <section>
+                    <h2>Bulk Import CLI</h2>
+                    Two files are required to describe an import
+                    <ul>
+                        <li>How to import: <b>bulk.yml</b></li>
+                    </ul>
+<pre style="font-size:55%"><code>
+continue: "true"
+transfer: "ln_s"
+checksum_algorithm: "File-Size-64"
+logprefix: "logs/"
+output: "yaml"
+path: "idr0021.tsv"
+columns:
+    - target
+    - path
+</code></pre>
+                        <ul>
+                            <li>What to import: <b>idr0021.tsv</b></li>
+                        </ul>
+<pre style="font-size:40%; overflow: scroll"><code>
+Dataset:name:CDK5RAP2-C /OMERO/in-place-import/idr0021-lawo-pericentriolarmaterial/Raw-files/CDK5RAP2-C/Centrin_PCNT_Cep215_20110506/Centrin_PCNT_Cep215_20110506_Fri-1545_0_SIR_PRJ.dv
+Dataset:name:CDK5RAP2-C /OMERO/in-place-import/idr0021-lawo-pericentriolarmaterial/Raw-files/CDK5RAP2-C/Centrin_PCNT_Cep215_20110506/Centrin_PCNT_Cep215_20110506_Fri-1559_0_SIR_PRJ.dv
+...
+</code></pre>      
+                </section>
+                <section>
+                    <h2>Post Import Data Manipulation: Demo...</h2>
+                    <ul><li>CLI: Render data
+                    <ul>
+                        <li>What to render: <i>renderingMapping.tsv</li>
+                        <pre style="font-size:40%; overflow: scroll"><code>
+CDK5RAP2-C_     renderingdef.yml
+CENT2_  renderingdef2.yml
+CEP120/20111106_        renderingdef.yml
+CEP120/20111209_        renderingdef2.yml
+...
+                        </code></pre>
+                        <li>How to render: <i>renderingdef.yml</li> & <i>renderingdef2.yml</li>
```

Comment:
This whole section renders as italics. I think it should be normal, not italics, except the filenames.